cypress/tsconfig.json

```diff
@@ -1,10 +1,7 @@
-// https://docs.cypress.io/guides/tooling/typescript-support#Types-for-plugins
```

Comment:
Not sure if removing the comments would be the proper fix tho. `tsconfig.json` file allows writing JSON5 format, which admits comments.

Can't we tell this CodeQL thing that those comments are fine?